shards/modules/gfx/feature.cpp

```diff
@@ -684,8 +685,9 @@ struct FeatureShard {
         .renderer = &ctx.renderer,
         .render = [&ctx = ctx](ViewPtr view, const PipelineSteps &pipelineSteps) { ctx.render(view, pipelineSteps); },
     };
-    mesh->variables.emplace(GraphicsRendererContext::VariableName,
-                            Var::Object(&graphicsRendererContext, GraphicsRendererContext::Type));
+
+    auto &grc = mesh->getVariable(ToSWL(GraphicsRendererContext::VariableName));
```

Comment:
@guusw this likely should be `mesh->addRef`

Reply:
or maybe not cos refs are released

Reply:
Should be good